achievements/tests.py

```diff
@@ -1,3 +1,75 @@
-from django.test import TestCase
+from django.test import TestCase, Client
+from .serializers import AchievementSerializer, CategorySerializer
+from .models import Achievement, Category
+from membership_file.models import Member
+from membership_file.serializers import MemberSerializer
 
 # Create your tests here.
+class TestCaseAchievementFrontEndViews(TestCase):
+    @classmethod
+
+    def setUp(self):
+        # This runs 1x per testcase (before) within this class
+        self.client = Client()
+
+        # Make a Category
+        self.categoryData = {
+            "name": "ZG-Category",
+            "description": "Boardgames-description",
+        }
+        self.category = Category.objects.create(**self.categoryData)
+
+        # Save the model
+        Category.save(self.category)
+
+        # Make an Achievement
+        self.achievementData = {
+            "category": Category.objects.all().first(),
+            "name": "Achievement_name",
+            "description": "Achievement_description",
+        }
+        self.achievement = Achievement.objects.create(**self.achievementData)
+        Achievement.save(self.achievement)
+
+        # Make a Member
+        self.memberData = {
+            "first_name": "Fantasy",
+            "last_name": "Court",
+            "date_of_birth": "1970-01-01",
+            "email": "info@fantasycourt.nl",
+            "street": "Veld",
+            "house_number": "5",
+            "city": "Eindhoven",
+            "country": "The Netherlands",
+            "postal_code": "5612 AH",
+            "member_since": "1970-01-01",
+        }
+        self.member = Member.objects.create(**self.memberData)
+        Member.save(self.member)
+
+    def test_case_view_all_categories(self):
```

Comment:
Missing a description of what the testcase tests. E.g. `#Tests if the page that shows all the achievement-categories can be accessed`